app/models/post.rb

```diff
@@ -5,6 +5,11 @@ class Post < ApplicationRecord
 
   after_save :update_post_user_counter
 
+  validates :title, presence: true
+  validates :text, length: { maximum: 250 }
+  validates :comments_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  validates :likes_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
```

Comment:
- [ ] Nice job :+1:  on putting validation, however you need to give these two columns a `0` default value in the database table so that the post may be created without an error. currently, I cannot create any post because their default value is null. Kindly update your migration by giving these columns `0` as the default value or initialize them with `0` ewne a post is created. 

![image](https://user-images.githubusercontent.com/65068771/161168266-d35ec5a1-ce2b-4e52-b0bc-1134af20197e.png)